DotNet-Platform.md

```diff
@@ -0,0 +1,40 @@
+![Logo][logo]
+
+# The .NET Platform 
+
+**.NET platform** is big and comprehensive collection of frameworks and libraries manager by [Microsoft](https://www.microsoft.com/net). In the beginning it could be quite daunting to get familiar with .Net ecosystem.
+
+Our aim is to get you familiar with the .Net platform, so that you are ready to learn the [c# programing language](C-Sharp) along with development environment, project structures, techniques and best practices you will be performing as a professional developer.
```

Comment:
Wouldn't it be the other way round? That they first learn C# to get familiar with .NET ecosystem? I am not sure which one of these two is correct, but I would ask that you at least ruminate over this sentence for a moment.